README.md

```diff
@@ -31,8 +31,12 @@ cd spectroscope
 python3 -m venv venv
 source venv/bin/activate
 
+# update the spectroscope-proto
+git submodules update --init 
+./build_proto_lib.sh
+
 # you can pull in dependencies for extra modules as necessary
-pip install .[alerta,webhook,zenduty]
+pip install .[alerta,webhook,zenduty] (zsh: pip install .\[alerta,webhook,zenduty\])
```

Comment:
I think you can `pip install ".[alerta,webhook,zenduty]"` and it will work on any shell

Reply:
Thanks for the reply! 
1. Just to be sure, you are talking about this test right? https://github.com/neukind/spectroscope/runs/3161402890?check_suite_focus=true
If that's the case yes it's totally out of date as now it's running in async, and the project has extra dependencies (mongodb related libraries: pymongo, motor). I will try to update it asap ! 
2. The MongoDB dependency is used for the Mongodb Change stream feature that allows me to trigger actions using the [asyncio.Queue() ](https://docs.python.org/3/library/asyncio-queue.html#queues) . 
Here is where I use the asyncio.queue to publish messages from the stream changes: 
https://github.com/HikaruG/spectroscope/blob/06514a7cd47f46d4a59b3bd2d2ca411db30925d8/spectroscope/clients/mongodb_client.py#L78